app/presenters/user_presenter.rb

```diff
@@ -0,0 +1,31 @@
+class UserPresenter < SimpleDelegator
+  include ActionView::Helpers::FormOptionsHelper
+
+  delegate :global, to: :user_roles, prefix: true
+
+  def self.wrap(users)
+    users.map { |user| new(user) }
+  end
+
+  def global_role_list
+    global_roles.join(", ")
+  end
+
+  def global_role_select_options
+    options_for_select(UserRole.global_roles, selected: user_roles.map(&:role))
+  end
+
+  def name_last_comma_first
```

Comment:
Why are we displaying last, first? I don't think we do that anywhere else on the site.